blockbase/block-template-parts/footer.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:group -->
 <div class="wp-block-group">
 	<!-- wp:paragraph {"align":"center","fontSize":"tiny"} -->
-	<p class="has-text-align-center has-tiny-font-size">Proudly Powered by WordPress</p>
+	<p class="has-text-align-center has-tiny-font-size">Proudly Powered by <a href="https://wordpress.com" rel="nofollow">WordPress</a></p>
```

Comment:
From looking at the diff, I believe this can also be a link to wordpress.org, which I would prefer since it means we can also submit the same code to the dotorg repo.

Reply:
Ah good call. Yes, the regex supports .org as well. I'll change. Thanks!